paymentsheet/src/main/java/com/stripe/android/link/account/DefaultLinkAccountManager.kt

```diff
@@ -349,12 +361,17 @@ internal class DefaultLinkAccountManager @Inject constructor(
     ): LinkAccount? {
         return lookup.consumerSession?.let { consumerSession ->
             if (startSession) {
-                setAccountNullable(
+                setAccount(
                     consumerSession = consumerSession,
                     publishableKey = lookup.publishableKey,
+                    displayablePaymentDetails = lookup.displayablePaymentDetails,
```

Comment:
store the default payment details info in the link account when coming on the lookup call.